<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPictureContrast</class>
 <widget class="QFrame" name="CPictureContrast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>678</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>910</width>
    <height>678</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>910</width>
    <height>678</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图像对比</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>车辆图片</string>
   </property>
   <widget class="QGraphicsView" name="gvVechile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>车主图片</string>
   </property>
   <widget class="QGraphicsView" name="gvOwner">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>进入图像</string>
   </property>
   <widget class="QLabel" name="lblEnterImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>401</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>离开图像</string>
   </property>
   <widget class="QLabel" name="lblEnterImg_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>289</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2011-04-18 12：01：02</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>选择费率</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>282</width>
       <height>44</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rdx1">
          <property name="text">
           <string>大型车</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdx2">
          <property name="text">
           <string>小型车</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdx3">
          <property name="text">
           <string>摩托车</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdx4">
          <property name="text">
           <string>自定义二</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rdx5">
          <property name="text">
           <string>中型车</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdx6">
          <property name="text">
           <string>免费车</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdx7">
          <property name="text">
           <string>自定义一</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdx8">
          <property name="text">
           <string>自定义三</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>891</width>
     <height>171</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btnPass">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>放行</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>卡号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtCardID"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>卡片自编号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtCardNumber"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>卡类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtCardType"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>车型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtVechileType"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>车牌：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtVechileID"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>171</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>车主：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtOwner"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>余额：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtRemainder"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>到期时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtExpiration"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>181</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>进入通道：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtEnterChannel"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>进入时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtEnterTime"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>离开通道：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtLeaveChannel"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>离开时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtLeaveTime"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>备注：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtComment"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>203</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>停留时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtStayTime"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>应收费：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spCharge"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnImg1">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImg2">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>60</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImg3">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>100</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImg4">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>150</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
